src/components/claimLink/claim.tsx

```diff
@@ -174,7 +174,7 @@ export function Claim() {
                     setLinkState('ALREADY_CLAIMED')
                 } else {
                     const _tokenprice = await utils.fetchTokenPrice(linkDetails.tokenAddress, linkDetails.chainId)
-                    setTokenPrice(_tokenprice)
+                    setTokenPrice(_tokenprice?.price.toString())
```

Comment:
What was the issue here before?

Reply:
No issue, just changes in the utils function